documentation/URL_SPEC.md

```diff
@@ -0,0 +1,159 @@
+
+You can include none or all of these parameters in the URL. If a parameter is
+not specified, the listed default value will be used instead.
+
+If you are using our public build, set
+[`https://volumeviewer.allencell.org/viewer`](https://volumeviewer.allencell.org/viewer)
+as the base address. If you are running Vol-E locally, you can substitute this
+for a `localhost` address.
+
+## Common Examples
+
+| Query Parameters                            | Description                                                                                                                               |
+| ------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `?url={url}&c0=ven:1&c2=ven:1,clz:1&view=Z` | Enable the first and third channel volumes and apply colorizing to the third. View in XY / Z-slice mode.                                  |
+| `?url={url}&c0=ven:1,rmp:15:180`            | Enables the volume of the first channel and ramps opacity for volume intensity values from 0% at histogram bin `15` to 100% at bin `180`. |
+| `?url={url}&reg=0:1,0:1,0.25:0.75,t=40`     | Clips along the Z-axis to the middle 50% of the volume, and sets the time to frame 40.                                                    |
+
+## Data source (`url`)
+
+The `url` parameter specifies the HTTPS URL of one or more volume to be loaded.
+Supported formats include OME-Zarr and OME-TIFF files.
+
+Multiple volumes can be loaded by including commas between each URL. This
+requires all volumes to have **some resolution/scale where the dimensions
+match.** The viewer will throw an error if there is no match possible.
+
+URLs containing special characters (`?`, `#`, `&`, or `,`) must be first encoded
+using
+[`encodeURIComponent`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent).
+
+| Query Parameters                | Description                               | Example                                                                                      |
+| ------------------------------- | ----------------------------------------- | -------------------------------------------------------------------------------------------- |
+| `?url={url}`                    | Load a volume from the specified URL.     | `?url=https://example.com/data/example1.ome.zarr`                                            |
+| `?url={url1},{url2},{url3},...` | Load multiple volumes from multiple URLs. | `?url=https://example.com/data/example1.ome.zarr,https://example.com/data/example2.ome.zarr` |
+
+## View settings
+
+General global settings for the viewer.
+
+| Query parameter | Description                                                                                                             | Expected Type                                                       | Default       | Example                         |
```

Comment:
"Expected Values" might be clearer here - I'd expect "type" to read to people like "string" or "number," thinking of types as subsets beneath those "classical" types (as TypeScript lets you do with e.g. literal types) is relatively niche.